Microsoft.Toolkit.Uwp.UI.Controls/SlidableListItem/SlidableListItem.cs

```diff
@@ -249,50 +267,104 @@ private void ContentGrid_ManipulationDelta(object sender, ManipulationDeltaRoute
                 return;
             }
 
-            _transform.TranslateX += e.Delta.Translation.X;
-            var abs = Math.Abs(_transform.TranslateX);
+            var abs = Math.Abs(_transform.TranslateX + e.Delta.Translation.X);
 
-            if (_transform.TranslateX > 0)
-            {
-                if (_commandContainer != null)
+            if (IsRightSwipeEnabled && e.Delta.Translation.X > 0)
+            { // Swiping from left to right.
+                if (_commandContainer != null && _transform.TranslateX > 0)
                 {
                     _commandContainer.Background = LeftBackground as SolidColorBrush;
+                    _leftCommandPanel.Opacity = 1;
+                    _rightCommandPanel.Opacity = 0;
                 }
 
-                _leftCommandPanel.Opacity = 1;
-                _rightCommandPanel.Opacity = 0;
-
-                if (abs < ActivationWidth)
```

Comment:
Actually it was something that I deleted deleted for debug purposes. If I add that, there has to be more work onto this commit because it's messing with the content that users see.

I will update it soon, thanks for pointing out.